packages/js/src/ui/components/Inbox.tsx

```diff
@@ -14,8 +14,8 @@ import { Button, Popover } from './primitives';
 
 export type InboxProps = {
   open?: boolean;
-  mountNotification?: NotificationMounter;
-  mountBell?: BellMounter;
+  renderNotification?: NotificationRenderer;
+  renderBell?: BellRenderer;
```

Comment:
⚠️ In the JS level, this is a mount method, not a render method. I think we should keep the names as they were. This also makes more sense to be explored in a new PR, or separate discussion.